### ISSUE TITLE
How to handle nested scrolling in Swiper

### ISSUE THREAD

**Issue by zsky1250** [nolimits4web/Swiper#1710]
I put a div witch contain a long content thus need scrolling in Swiper
like this

**div id="slide2" class="swiper-slide"
*****div id="content" style=" width:400px;height:400px;overflow-y: scroll; "
***********there is a lot of content,actually it's a shopping list here,so I need scrolling the div.
***********at the same time,I need scrolling swiper too!
        ......

I konw it's not a smart design,but customer is the GOD...
I have read some issue about this problem.Could you give me a solution about it.
or Could I get a chance to preventing Swiper's default scrolling listening and adding it to my div at a fix region?

**Comment by navyxie** [comment]
mybe,you can lock the swiper when the div scrolling,and unlock when the div scroll end.

**Comment by zsky1250** [comment]
Finally found a solution myself!
I disabled the 'longSwipes' feature and give it to my scrolling div.
In details:
1. define options of swipes:    
    longSwipes: false,//to disabled and 
   longSwipesMs : 300// setting the time threshold between long and short. 
2.onSlideChangeEnd add some logic : setting the followFinger feature to false when current slide containing Although moving slide with finger is good.when I want to scroll my div,it's a big trouble. 
3.onTouchMove & onTouchEnd：write my scrolling logic as a callback.
As a result.when you quick swiping it will scrolling the div content,long swiping will make page sliding.

Thanks you, you have made a great lib,Not even the optimized also about the extension.
when I writing my logic,I used swiper.params.xxx and swiper.touches.diff, it's helps a lot.

Hoping can make some help to further developing of Swiper.js
